e2e-tests/src/staking.rs

```diff
@@ -97,3 +100,107 @@ pub fn wait_for_era_completion(
     wait_for_session(connection, first_session_in_next_era)?;
     Ok(next_era_index)
 }
+
+pub fn check_non_zero_payouts_for_era(
+    node: &String,
+    stash: &KeyPair,
+    connection: &Connection,
+    era: BlockNumber,
+) {
+    let stash_account = AccountId::from(stash.public());
+    let locked_stash_balance_before_payout = locks(&connection, &stash);
+    assert!(
+        locked_stash_balance_before_payout.is_some(),
+        "Expected non-empty locked balances for account {}!",
+        stash_account
+    );
+    let locked_stash_balance_before_payout = locked_stash_balance_before_payout.unwrap();
```

Comment:
why not to use `expect`?

Reply:
Will fix

Reply:
I meant that the whole fragment (7 lines) can be squashed to a single `expect`

Reply:
Okay, makes sense, will fix.